spec/features/posts_feature_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+feature 'posts' do
+
+  context 'no posts have been added' do
+
+    scenario 'should display a prompt to add a post' do
+      visit '/posts'
+      expect(page).to have_content 'No posts yet'
+      expect(page).to have_link 'Post to Instagram'
+    end
+  end
+
+  context 'creating posts' do
+
+    scenario 'can post photos' do
+      user = FactoryGirl.create(:user)
+      login_as(user, :scope => :user)
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)